language/dotnet/develop.md

```diff
@@ -0,0 +1,460 @@
+           "Microsoft": "Warning",
+           "Microsoft.Hosting.Lifetime": "Information"
+       }
+   },
+   "AllowedHosts": "*",
+   "ConnectionStrings": {
+       "SchoolContext": "Host=db;Database=my_db;Username=postgres;Password=example"
+   }
+}
+```
+
+Save and close the `appsettings.json` file.
+
+In the `Index.cshtml` file located in the `src\Pages` directory, replace the contents with the following code:
+
+```html
+@page
+@model IndexModel
+@{
+    ViewData["Title"] = "Home page";
+}
+
+<div class="text-center">
+    <h1 class="display-4">Welcome</h1>
+    <p>Learn about <a href="https://docs.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p>
+</div>
+
+<div class="row mb-auto">
+    <p>Student Name is @Model.StudentName</p>
+</div>
+```
+
+Save and close the `Index.cshtml` file.
+
+In the `Index.cshtml.cs` file located in the `src\Pages` directory, replace the contents with the following code:
+
+```c#
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace myWebApp.Pages;
+
+public class IndexModel : PageModel
+    {
+        public string StudentName { get; private set; } = "PageModel in C#";
+        private readonly ILogger<IndexModel> _logger;
+        private readonly myWebApp.Data.SchoolContext _context;
+
+        public IndexModel(ILogger<IndexModel> logger, myWebApp.Data.SchoolContext context)
+        {
+            _logger = logger;
+            _context= context;
+        }
+
+        public void OnGet()
+        {
+            var s =_context.Students?.Where(d=>d.ID==1).FirstOrDefault();
+            this.StudentName = $"{s?.FirstMidName} {s?.LastName}";
+        }
+    }
+```
+
+Save and close the `Index.cshtml.cs` file.
+
+Now we can rebuild our image. Open a terminal, change directory to the `dotnet-docker` directory and run the following command:
+
+```console
+$ docker build --tag dotnet-docker .
+```
+
+List your running containers.
+
+```console
+$ docker ps
+CONTAINER ID   IMAGE           COMMAND                  CREATED          STATUS          PORTS                            NAMES
+146e1cb76e71   postgres        "docker-entrypoint.s…"   25 minutes ago   Up 25 minutes   5432/tcp                         postgresqldb
+72bef28b1cd4   dotnet-docker   "./myWebApp"             40 minutes ago   Up 40 minutes   0.0.0.0:5000->80/tcp             dotnet-app
+```
+Inspect the image column and stop any container that is using the `dotnet-docker` image.
+
+```console
+$ docker stop dotnet-app
+dotnet-app
+```
+
+Now, let’s run our container on the same network as the database. This allows us to access the database by its container name.
+
+```console
+$ docker run \
+  --rm -d \
+  --network postgres-net \
+  --name dotnet-app \
+  -p 5000:80 \
+  dotnet-docker
+```
+
+Let's test that the application works and is connecting to the database. Using a web browser, access `http://localhost:5050`. A page similar to the following image appears.
```

Comment:
It's port 5000, not 5050, there also are some similar differences further down this page.

Reply:
Thank you. Will update this occurrence to 5000.